pkg/services/ngalert/notifier/external_alertmanager.go

```diff
@@ -267,3 +280,33 @@ func (r *roundTripper) RoundTrip(req *http.Request) (*http.Response, error) {
 
 	return r.next.RoundTrip(req)
 }
+
+// TODO: change implementation, this is only useful for testing other methods.
```

Comment:
I think we already have an issue for this, could you link it on the PR? 🙏

Reply:
It's not a public one :(